libraries/Microsoft.Bot.Builder.LanguageGeneration/MultiLanguageLG.cs

```diff
@@ -59,7 +59,7 @@ public MultiLanguageLG(Dictionary<string, Templates> templatesPerLocale, string
         /// <param name="data">Scope data.</param>
```

Comment:
>data [](start = 25, length = 4)

Why not call this scope like other APIs like Templates.Evaluate?

Reply:
At the very beginning MultilanguageLG implements ILanguageGenerator interface which has the method "generate". But now there is no such dependency, we can consider changing the name.
@boydc2014 any thougths?

Reply:
I don't have naming preference, i think keep the name "Generate" is good to me, we don't have to introduce name change here, @vishwacsena  what do you think?

Reply:
Yes. Do not believe we need a method name change here.